libcloud/common/ovh.py

```diff
@@ -115,17 +116,23 @@ def request_consumer_key(self, user_id):
 
     def get_timestamp(self):
         if not self._timedelta:
-            url = 'https://%s/%s/auth/time' % (API_HOST, API_ROOT)
+            url = 'https://%s%s/auth/time' % (API_HOST, API_ROOT)
```

Comment:
Because `API_ROOT` already starts with `'/'` (https://github.com/apache/libcloud/pull/891/files/a5ed7e1f907cd1fcb5dde3f873bf4855012f3fe7#diff-b98d1474f4f20fc646a3ac82fcbd0387R37) . So it will make a doublet.
Maybe it worked with RunAbove, but not with OVH